IBPSA/Electrical/Data/PV/Generic.mo

```diff
@@ -0,0 +1,37 @@
+within IBPSA.Electrical.Data.PV;
+record Generic "Basic record of a PV cell"
+  extends Modelica.Icons.Record;
+
+  parameter Integer nSer "Number of cells connected in series on the PV panel"
+    annotation(Dialog(group="General"));
+  parameter Integer nPar "Number of parallel cell circuits on the PV panel"
+    annotation(Dialog(group="General"));
+  parameter Modelica.Units.SI.Area ACel
+    "Area of a single cell. If not found in data sheet, use ACel = ((V_mp0*I_mp0)/(1000*eta_0))/n_ser"
```

Comment:
Comment is not clear, where should `V_mp0` (and other parameters) be obtained? Also, I think you mean `nSer`.
Maybe such instructions would be better in the info section.

Reply:
I deleted it